Implement FastADT experiment (#137)
instamatic-dev/instamatic: docs/config.md

**interface**
: Give the interface of the camera interface to connect to, for example: `timepix`/`emmenu`/`simulate`/`gatan`/`merlin`. Leave blank or set to `None` to load the camera specs, but do not load the camera module (this also turns off the videostream gui).

**dead_time**

> : Set the dead time (i.e. the gap between acquisitions) of the detector; if absent and needed to collect a movie, an attempt to use a `CalibMovieDelays.dead_time` value calibrated via `instamatic.calibrate_movie_delays` will be made.

Reviewer: Small attempt to make the text a bit clearer.

```suggestion
: Set the dead time (i.e. the gap between acquisitions) of the detector. If this value (`CalibMovieDelays.dead_time`) is not set and needed to collect a movie, Instamatic will attempt to calibrate the value via `instamatic.calibrate_movie_delays`.
```
